fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -676,6 +700,7 @@ public void readFields(DataInput in) throws IOException {
         Text.readString(in);
         dbState = DbState.valueOf(Text.readString(in));
         attachDbName = Text.readString(in);
+        dbComment = Text.readString(in);
```

Comment:
read comment only if version >= 132

Reply:
Thank you for your suggestion. I will revise it.